### ISSUE TITLE
Outputting the Context separately from the generated changelog

### ISSUE THREAD

**Issue by leonzalion** [orhun/git-cliff#71]
**Is your feature request related to a problem? Please describe.**
git-cliff looks amazing for parsing git commits, but sometimes I'd love to have more power over the parsed commit messages (i.e. the [Context](https://github.com/orhun/git-cliff#context) that's passed to Tera) instead of being forced to use a Tera template to generate my changelog.

**Describe the solution you'd like**
Would it be possible to have an option added to git-cliff that would output the parsed commits as a normal JSON file without first passing it through a Tera template?

**Comment by orhun** [comment]
Hello!

Thank you for the nice suggestion. I pushed [95ad55d](https://github.com/orhun/git-cliff/commit/95ad55d542cb772881f12ae2a25f7e06202c8587) which adds `--context` flag for outputting the changelog context as you described:

```sh
$ git cliff --context | jq .
$ git cliff --context --output context.json
```

This flag will be available with the upcoming release so feel free to test this out and report bugs (if any) until then!

**Comment by leonzalion** [comment]
Perfect, it works beautifully, thank you so much! One question I have is whether the warning "git_cliff > Changelog body is not specified, using the default template" should still be displayed when the `--context` flag is used. If I understand correctly, the `body` tag should be redundant when using `--context` (Correct me if I'm wrong!)

**Comment by orhun** [comment]
> Perfect, it works beautifully, thank you so much! One question I have is whether the warning "git_cliff > Changelog body is not specified, using the default template" should still be displayed when the `--context` flag is used. If I understand correctly, the `body` tag should be redundant when using `--context` (Correct me if I'm wrong!)

Ah, good catch. Fixed it in efd40e02b3c7be29da6e2a161423b92a74c4f46e
